docs/docs/deploying-to-now.md

```diff
@@ -1,80 +1,31 @@
 ---
-title: Deploying to Now
+title: Deploying to Vercel Now
 ---
 
-[ZEIT Now](https://zeit.co/now) is a [cloud platform for serverless deployment](https://zeit.co/docs/v2/getting-started/introduction-to-now/) that you can use to deploy your Gatsby projects and [alias them](https://zeit.co/docs/v2/domains-and-aliases/aliasing-a-deployment/) to your personal domain or a free `.now.sh` suffixed URL.
+[Vercel Now](https://vercel.com/now) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gatsby projects to your personal domain (or a free `.now.sh` suffixed URL).
 
 This guide will show you how to get started in a few quick steps:
 
 ## Step 1: Getting Started with Gatsby
```

Comment:
This section still seems to have been ripped out. Step 1 was previously for setting up a gatsby site and you immediately jump into setting up the now CLI